mne/preprocessing/ica.py

```diff
@@ -899,7 +909,7 @@ def _export_info(self, info, container, add_channels):
             # re-append additionally picked ch_info
             ch_info += [k for k in container.info['chs'] if k['ch_name'] in
                         add_channels]
-        info['bads'] = [ch_names[k] for k in self.exclude]
+        info['bads'] = [ch_names[k] for k in list(self.exclude)]
```

Comment:
Why is this necessary?

Reply:
See here: #5967

Reply:
I don't see it.

Reply:
It was again eating my (2nd) comment ... Yes, you are right, not necessary here.